src/SpaceWatcher.ts

```diff
@@ -10,18 +10,19 @@ import { APP_PLAYLIST_REFRESH_INTERVAL } from './constants/app.constant'
 import { TWITTER_AUTHORIZATION } from './constants/twitter.constant'
 import { Downloader } from './Downloader'
 import { logger as baseLogger } from './logger'
-import { SpaceChat } from './SpaceChat'
+import { Twitter } from './namespaces/Twitter'
+import { SpaceCaptions } from './SpaceCaptions'
```

Comment:
Shouldn't this be in some `interfaces` folder or something

Reply:
It includes interfaces/enums so better have separate folder 🤔